docs/userguide/parameters.md

```diff
@@ -127,11 +136,43 @@ SubstanceParameter(
 )
 ```
 
-The ``encoding`` option defines what kind of descriptors are calculated:
-* ``MORDRED``: 2D descriptors from the [Mordred package](https://mordred-descriptor.github.io/documentation/master/).
-  Since the original package is now unmaintained, baybe requires the community replacement [mordredcommunity](https://github.com/JacksonBurns/mordred-community)
-* ``RDKIT``: 2D descriptors from the [RDKit package](https://www.rdkit.org/)
-* ``MORGAN_FP``: Morgan fingerprints calculated with RDKit (1024 bits, radius 4)
+The ``encoding`` option defines what kind of descriptors are calculated using the
+[scikit-fingerprints] package.
+Any of the fingerprint classes provided by the package can be used.
```

Comment:
Careful: If they add more classes, are these then directly supported by us? If I understand this PR correctly, this would not be the case and would require us to update our encodings, right? So this is a statement that might become incorrect at any point in time.

Reply:
this is true, would suggest to just remove the sentence